MSALiOS/ViewController.swift

```diff
@@ -33,196 +33,232 @@ import MSAL
 class ViewController: UIViewController, UITextFieldDelegate, URLSessionDelegate {
     
     // Update the below to your client ID you received in the portal. The below is for running the demo only
-    
     let kClientID = "2a814505-ab4a-41f7-bd09-3fc614ac077c"
     
     // These settings you don't need to edit unless you wish to attempt deeper scenarios with the app.
-    
     let kGraphURI = "https://graph.microsoft.com/v1.0/me/"
     let kScopes: [String] = ["https://graph.microsoft.com/user.read"]
-    let kAuthority = "https://login.microsoftonline.com/common/"
+    let kAuthority = "https://login.microsoftonline.com/common"
     
     var accessToken = String()
-    var applicationContext = MSALPublicClientApplication.init()
-    
+    var applicationContext : MSALPublicClientApplication?
 
     @IBOutlet weak var loggingText: UITextView!
     @IBOutlet weak var signoutButton: UIButton!
+
+    /**
+        Setup public client application in viewDidLoad
+    */
+
+    override func viewDidLoad() {
+
+        super.viewDidLoad()
+
+        do {
+
+            /**
+
+             Initialize a MSALPublicClientApplication with a given clientID and authority
+
+             - clientId:            The clientID of your application, you should get this from the app portal.
+             - authority:           A URL indicating a directory that MSAL can use to obtain tokens. In Azure AD
+                                    it is of the form https://<instance/<tenant>, where <instance> is the
+                                    directory host (e.g. https://login.microsoftonline.com) and <tenant> is a
+                                    identifier within the directory itself (e.g. a domain associated to the
+                                    tenant, such as contoso.onmicrosoft.com, or the GUID representing the
+                                    TenantID property of the directory)
+             - error                The error that occurred creating the application object, if any, if you're
+                                    not interested in the specific error pass in nil.
+             */
+
+            self.applicationContext = try MSALPublicClientApplication.init(clientId: kClientID, authority: kAuthority)
```

Comment:
nit: we can skip `.init` here, just write: `MSALPublicClientApplication(clientId: kClientID, authority: kAuthority)`

Reply:
yeah, I saw it today, it was copied from the old code :) Will fix.